### PR TITLE
Acl postwork

### PR DESCRIPTION
**Pull request by g-krisztian**
Reacting to framework and template changes.

### CODE REVIEW

File: src/framework/components/app/core.clj [hunk 1]
```diff
@@ -65,22 +66,25 @@
...
-
            (fn [http-request]
              (->
                (apply m/>>=
                  (concat
                    [(xiana.core/ok (create-empty-state))
                     (fn [x] (add-deps x {:router router, :db db}))
-                    (fn [x] (add-http-request x http-request))]
+                    (fn [x] (add-http-request x http-request))
+                    (fn [this] (init-acl this acl-cfg))]
```
Comment (defalut) [inline src/framework/components/app/core.clj:87]: you can access `acl-cfg` using `(-> this :acl-cfg)`, so no need for acl-cfg to be a component

File: src/framework/components/app/core.clj [hunk 2]
```diff
@@ -65,22 +66,25 @@
...
        (map #(fn [x] (% x)))))
 
-(defrecord App
-  [config router db]
+(defn init-acl
+  [this config]
+  (acl-builder/init this config))
 
+(defrecord App
+  [config acl-cfg router db]
```
Comment (defalut) [inline src/framework/components/app/core.clj:74]: [config router db]
you can dig acl-cfg out of config

File: src/framework/components/app/core.clj [hunk 3]
```diff
@@ -91,5 +95,5 @@
                (get :response))))))
 
 (defn make-app
-  [config router-interceptors controller-interceptors]
-  (map->App {:config config :router-interceptors router-interceptors :controller-interceptors controller-interceptors}))
+  [config acl-cfg router-interceptors controller-interceptors]
+  (map->App {:config config :acl-cfg acl-cfg :router-interceptors router-interceptors :controller-interceptors controller-interceptors}))
```
Comment (defalut) [inline src/framework/components/app/core.clj:99]: you do not need to pass `acl-cfg` if it is `(select-keys config [:acl/permissions :acl/roles])`

**Comment by LeaveNhA** [comment]
It's already done in ACL.

**Pull request closed without merging**